tests/secure_tunneling_tests.c

```diff
@@ -47,6 +47,10 @@ struct secure_tunneling_test_context {
 };
 static struct secure_tunneling_test_context s_test_context = {.max_threads = 1};
 
+/* Dummy websocket for  */
```

Comment:
is this sentence incomplete? 

Everything else looks good to me.

Reply:
Fixed in the next revision.